examples/deploy-agent/README.md

```diff
@@ -29,9 +29,13 @@ This sample shows how to deploy a trained agent using ``ray``'s module
     - Change the variable ``CHECKPOINT_FOLDER`` to the name of the folder containing your RLlib checkpoints.
     - Please note that ``CHECKPOINT_FOLDER`` must be the name of the folder inside ``checkpoints`` that contains the trained agent.
     - Modify the `observation_space` and `action_space` variables to be the same as in your simulation environment used for training the agent.
```

Comment:
We can remove everything related to modifying `observation_space` and `action_space` if we're registering the sim. This info will be read from the sim.

Reply:
I actually went the route of creating a dummy env with the given spaces. In this way the user does not have to copy-paste the sim.
Check the ``restore.py`` file in ``platotk``.